src/commands/paranet/paranet-sync-command.js

```diff
@@ -0,0 +1,218 @@
+                    contract,
+                    tokenId,
+                    stateIndex,
+                    SIMPLE_ASSET_SYNC_PARAMETERS.STATUS.IN_PROGRESS,
+                ),
+
+                this.repositoryModuleManager.createOperationRecord(
+                    this.getService.getOperationName(),
+                    operationId,
+                    OPERATION_ID_STATUS.IN_PROGRESS,
+                ),
+            ]);
+
+            const hashFunctionId = CONTENT_ASSET_HASH_FUNCTION_ID;
+
+            this.logger.debug(
+                `ASSET_SYNC: Get for ${ual} with operation id ${operationId} initiated. blockchain: ${blockchain}`,
+            );
+
+            await this.commandExecutor.add({
+                name: 'networkGetCommand',
+                sequence: [],
+                delay: 0,
+                data: {
+                    operationId,
+                    id: ual,
+                    blockchain,
+                    contract,
+                    tokenId,
+                    state: assertionId,
+                    hashFunctionId,
+                    assertionId,
+                    assetSync: true,
+                    stateIndex,
+                    assetSyncInsertedByCommand: true,
+                    paranetSync: true,
+                    paranetId,
+                    paranetRepoId: newRepoId,
+                    paranetLatestAsset: latestAsset,
+                    paranetDeleteFromEarlier: deleteFromEarlier
+                },
+                transactional: false,
+            });
+
+            await this.operationIdService.updateOperationIdStatus(
+                operationId,
+                blockchain,
+                OPERATION_ID_STATUS.GET.GET_INIT_END,
+            );
+
+            let attempt = 0;
+            let getResult;
+            do {
+                await setTimeout(SIMPLE_ASSET_SYNC_PARAMETERS.GET_RESULT_POLLING_INTERVAL_MILLIS);
+                getResult = await this.operationIdService.getOperationIdRecord(operationId);
+                attempt += 1;
+            } while (
+                attempt < SIMPLE_ASSET_SYNC_PARAMETERS.GET_RESULT_POLLING_MAX_ATTEMPTS &&
+                getResult?.status !== OPERATION_ID_STATUS.FAILED &&
+                getResult?.status !== OPERATION_ID_STATUS.COMPLETED
+            );
+        } catch (error) {
+            this.logger.warn(
+                `ASSET_SYNC: Unable to sync tokenId: ${tokenId}, for contract: ${contract} state index: ${stateIndex} blockchain: ${blockchain}, error: ${error}`,
+            );
+            await this.repositoryModuleManager.updateAssetSyncRecord(
+                blockchain,
+                contract,
+                tokenId,
+                stateIndex,
+                SIMPLE_ASSET_SYNC_PARAMETERS.STATUS.FAILED,
+                true,
+            );
+        }
+    }
+
+    /**
+     * Recover system from failure
+     * @param command
+     * @param error
+     */
+    async recover(command) {
```

Comment:
Please double check if we need to pass command here as argument

Reply:
Other commands have it and it's used for logging